nmt_keras/dq_evaluation.py

```diff
@@ -45,14 +46,18 @@ def qe_metrics(pred_list, verbose, extra_vars, split, ds, set, no_ref=False):
             final_scores = eval_sent_qe(ref, sent_pred, 'Sent')
 
     elif set=='doc_qe':
-        
-        ref = eval("ds.Y_"+split+"['doc_qe']")        
+
+        ref = eval("ds.Y_"+split+"['doc_qe']")
         final_scores = eval_sent_qe(ref, pred_list[0], 'Doc')
 
 
     elif set=='word_qe':
-        ref = eval("ds.Y_"+split+"['word_qe']")
-        final_scores = eval_word_qe(ref, pred_list[0], ds.vocabulary['word_qe'], 'Word')
+        target_text = eval("ds.X_"+split+"['target_text']")
```

Comment:
Using `eval` like this is terrible (use `getattr` instead), and I'm only letting you use it here because it's inherited from `nmt-keras`. At some point, we'll clean it all out. But don't use it like this in other projects.